decidim-budgets/app/models/decidim/budgets/project.rb

```diff
@@ -36,6 +37,11 @@ def comments_have_votes?
       def confirmed_orders_count
         orders.finished.count
       end
+
+      # Public: Overrides the `notifiable?` Notifiable concern method.
+      def notifiable?(_context)
+        false
```

Comment:
Yeah I thought the same @deivid-rodriguez . I think the `Commentable` module can include the `Notifiable` but then I should rename it to something like `NotifiableOnComment` or something like that (I'm very bad at naming things).